server/sonar-web/src/main/js/app/components/nav/global/GlobalNavMenu.js

```diff
@@ -36,48 +37,53 @@ export default class GlobalNavMenu extends React.Component {
   }
 
   renderProjects () {
-    const controller = this.props.currentUser.isLoggedIn ? '/projects/favorite' : '/projects';
-    const url = window.baseUrl + controller;
+    const pathname = this.props.currentUser.isLoggedIn ? '/projects/favorite' : '/projects';
     return (
-        <li className={this.activeLink('/projects')}>
-          <a href={url}>{translate('projects.page')}</a>
+        <li>
+          <Link to={{ pathname }} activeClassName="active">
+            {translate('projects.page')}
+          </Link>
         </li>
     );
   }
 
   renderIssuesLink () {
+    // TODO use <Link/>
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=javascript%3AS1135)